src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Lottos {
+    private List<Lotto> lottos = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Lotto> lottos = new ArrayList<>();
```

가급적 부수효과가 적은 불변 객체 형태로 만들어 주세요 😄 

참고) https://velog.io/@conatuseus/Java-Immutable-Object%EB%B6%88%EB%B3%80%EA%B0%9D%EC%B2%B4